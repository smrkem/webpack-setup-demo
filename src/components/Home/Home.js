import React from 'react'
import PostSummary from '../PostSummary/PostSummary'

import posts from '../../posts.json'

const Home = () => (
  <div className="container">
    {posts.map(post => (
      <PostSummary key={post.slug} {...post} />
    ))}
  </div>
)

export default Home
