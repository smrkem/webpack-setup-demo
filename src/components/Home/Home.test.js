import React from 'react'
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from './Home'

test('Home should render ok', () => {
  const component = shallow(<Home />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
