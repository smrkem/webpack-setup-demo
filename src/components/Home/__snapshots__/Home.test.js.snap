// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render ok 1`] = `
<div
  className="container"
>
  <PostSummary
    author="Matt Smrke"
    content="Hammock locavore bespoke williamsburg. Organic adaptogen offal single-origin coffee tattooed prism tumeric. Listicle fixie polaroid tbh la croix man braid celiac. Vice flannel health goth chia, organic shaman truffaut kinfolk narwhal quinoa echo park messenger bag. Occupy vape disrupt, try-hard scenester food truck whatever aesthetic vexillologist. Kogi dreamcatcher authentic, selfies hella vaporware keffiyeh asymmetrical. Gluten-free hell of air plant polaroid, glossier bicycle rights salvia master cleanse jianbing post-ironic portland chillwave occupy. Chicharrones schlitz hell of hammock echo park health goth taxidermy migas tofu. Schlitz polaroid roof party, before they sold out sustainable coloring book yuccie craft beer chartreuse iceland. VHS unicorn sustainable jianbing, ethical lyft ugh plaid activated charcoal chicharrones semiotics blue bottle polaroid tbh tattooed. Lyft heirloom ennui, adaptogen yuccie slow-carb cloud bread succulents affogato pitchfork salvia. Pork belly unicorn crucifix adaptogen, ennui kinfolk letterpress. Locavore lumbersexual woke freegan paleo man braid lo-fi actually. Locavore man bun church-key offal.
  
  Roof party polaroid succulents, vice raw denim synth sartorial normcore locavore vinyl. VHS hella 8-bit godard, vape selfies humblebrag raclette cornhole pabst pickled bitters coloring book pitchfork pork belly. Four dollar toast next level hoodie roof party listicle enamel pin dreamcatcher vegan single-origin coffee bitters meggings pinterest yr tousled microdosing. Sustainable vaporware distillery synth portland. Semiotics freegan tumeric viral godard put a bird on it intelligentsia, kale chips listicle readymade bitters 90's tote bag coloring book. Photo booth celiac retro pitchfork PBR&B taiyaki raw denim asymmetrical prism. Fashion axe actually fixie asymmetrical pickled iceland 90's. Celiac direct trade you probably haven't heard of them leggings actually iceland tbh. Sartorial activated charcoal PBR&B vegan vape, sriracha next level humblebrag fanny pack chartreuse. Listicle cloud bread chartreuse seitan la croix. Pok pok farm-to-table church-key master cleanse disrupt pork belly authentic etsy art party wayfarers. Before they sold out bushwick small batch thundercats helvetica, butcher glossier chia offal gochujang forage 8-bit cold-pressed. Mustache vice tilde, kinfolk copper mug kombucha tbh cornhole letterpress chia cred thundercats beard post-ironic air plant."
    excerpt="Hammock locavore bespoke williamsburg. Organic adaptogen offal single-origin coffee tattooed prism tumeric. Listicle fixie polaroid tbh la croix man braid celiac. Vice flannel health goth chia, organic shaman truffaut kinfolk narwhal quinoa echo park messenger bag. Occupy vape disrupt, try-hard scenester food truck whatever aesthetic vexillologist. Kogi dreamcatcher authentic, selfies hella vaporware keffiyeh asymmetrical. Gluten-free hell of air plant polaroid, glossier bicycle rights salvia master cleanse jianbing post-ironic portland chillwave occupy."
    published_at="2017-10-22"
    slug="sample-post-1"
    title="Sample Post 1"
  />
  <PostSummary
    author="Matt Smrke"
    content="Farm-to-table authentic heirloom chicharrones. Before they sold out pour-over quinoa normcore stumptown helvetica gastropub. Deep v succulents mustache synth cold-pressed. Drinking vinegar vaporware next level PBR&B pinterest la croix seitan. Tousled blue bottle iPhone glossier franzen jean shorts street art shaman stumptown next level. Fingerstache synth organic, single-origin coffee kombucha vinyl pinterest brooklyn you probably haven't heard of them tbh before they sold out hot chicken 3 wolf moon chicharrones meditation. Bespoke activated charcoal you probably haven't heard of them pop-up drinking vinegar jean shorts tumblr occupy. Typewriter chicharrones tumeric heirloom hella, tousled sartorial polaroid 90's vegan neutra trust fund twee tattooed keffiyeh. Live-edge air plant hammock polaroid four dollar toast dreamcatcher kogi sriracha vexillologist pug.
  
  Leggings whatever vegan palo santo hoodie tote bag. Marfa lyft typewriter tofu hashtag mumblecore tumblr trust fund jean shorts migas. Tofu YOLO keytar messenger bag stumptown pabst selfies kombucha. Af shaman 8-bit chartreuse sriracha irony godard live-edge cred. Poke roof party activated charcoal forage. Live-edge actually umami leggings tousled. Bitters la croix ethical microdosing, air plant actually fixie ennui. Pour-over cardigan etsy four dollar toast, glossier cornhole flannel copper mug 3 wolf moon hammock pabst hot chicken keytar seitan. Taiyaki pok pok man bun street art intelligentsia biodiesel. Man bun locavore air plant hot chicken mixtape selvage ramps. Lomo marfa jianbing intelligentsia vice letterpress, tofu artisan retro tattooed.
  
  Etsy bitters DIY farm-to-table try-hard, jianbing waistcoat everyday carry leggings. Shabby chic cliche lo-fi literally green juice, live-edge street art letterpress enamel pin unicorn. Migas literally jean shorts, beard tumeric succulents edison bulb adaptogen deep v fixie twee fam chia keffiyeh intelligentsia. Cardigan everyday carry green juice jean shorts helvetica mustache lyft man braid enamel pin ramps polaroid venmo bespoke lomo hella. Stumptown crucifix brunch flexitarian shaman organic, etsy glossier slow-carb put a bird on it master cleanse blog seitan actually. Bespoke neutra scenester, glossier food truck authentic seitan. Craft beer artisan man braid viral. Vape selfies marfa lumbersexual mixtape celiac disrupt VHS selvage squid before they sold out farm-to-table blog. Coloring book bitters fashion axe semiotics typewriter banh mi chia cloud bread hella literally locavore vape fingerstache bespoke. Cornhole pinterest chicharrones, wolf helvetica kinfolk fam williamsburg poutine organic mlkshk iPhone franzen poke cold-pressed. Pug authentic vexillologist, squid brunch vaporware air plant fanny pack fixie dreamcatcher hot chicken tofu blog banh mi."
    excerpt="Farm-to-table authentic heirloom chicharrones. Before they sold out pour-over quinoa normcore stumptown helvetica gastropub. Deep v succulents mustache synth cold-pressed. Drinking vinegar vaporware next level PBR&B pinterest la croix seitan. Tousled blue bottle iPhone glossier franzen jean shorts street art shaman stumptown next level. Fingerstache synth organic, single-origin coffee kombucha vinyl pinterest brooklyn you probably haven't heard of them tbh before they sold out hot chicken 3 wolf moon chicharrones meditation."
    published_at="2017-10-22"
    slug="sample-post-2"
    title="Sample Post 2"
  />
  <PostSummary
    author="Matt Smrke"
    content="Roof party polaroid succulents, vice raw denim synth sartorial normcore locavore vinyl. VHS hella 8-bit godard, vape selfies humblebrag raclette cornhole pabst pickled bitters coloring book pitchfork pork belly. Four dollar toast next level hoodie roof party listicle enamel pin dreamcatcher vegan single-origin coffee bitters meggings pinterest yr tousled microdosing. Sustainable vaporware distillery synth portland. Semiotics freegan tumeric viral godard put a bird on it intelligentsia, kale chips listicle readymade bitters 90's tote bag coloring book. Photo booth celiac retro pitchfork PBR&B taiyaki raw denim asymmetrical prism. Fashion axe actually fixie asymmetrical pickled iceland 90's. Celiac direct trade you probably haven't heard of them leggings actually iceland tbh. Sartorial activated charcoal PBR&B vegan vape, sriracha next level humblebrag fanny pack chartreuse. Listicle cloud bread chartreuse seitan la croix. Pok pok farm-to-table church-key master cleanse disrupt pork belly authentic etsy art party wayfarers. Before they sold out bushwick small batch thundercats helvetica, butcher glossier chia offal gochujang forage 8-bit cold-pressed. Mustache vice tilde, kinfolk copper mug kombucha tbh cornhole letterpress chia cred thundercats beard post-ironic air plant.
  
  Farm-to-table authentic heirloom chicharrones. Before they sold out pour-over quinoa normcore stumptown helvetica gastropub. Deep v succulents mustache synth cold-pressed. Drinking vinegar vaporware next level PBR&B pinterest la croix seitan. Tousled blue bottle iPhone glossier franzen jean shorts street art shaman stumptown next level. Fingerstache synth organic, single-origin coffee kombucha vinyl pinterest brooklyn you probably haven't heard of them tbh before they sold out hot chicken 3 wolf moon chicharrones meditation. Bespoke activated charcoal you probably haven't heard of them pop-up drinking vinegar jean shorts tumblr occupy. Typewriter chicharrones tumeric heirloom hella, tousled sartorial polaroid 90's vegan neutra trust fund twee tattooed keffiyeh. Live-edge air plant hammock polaroid four dollar toast dreamcatcher kogi sriracha vexillologist pug."
    excerpt="Roof party polaroid succulents, vice raw denim synth sartorial normcore locavore vinyl. VHS hella 8-bit godard, vape selfies humblebrag raclette cornhole pabst pickled bitters coloring book pitchfork pork belly. Four dollar toast next level hoodie roof party listicle enamel pin dreamcatcher"
    published_at="2017-10-22"
    slug="sample-post-3"
    title="Sample Post 3"
  />
  <PostSummary
    author="Matt Smrke"
    content="Try-hard edison bulb slow-carb pitchfork tumeric yr master cleanse health goth actually cred shoreditch. Enamel pin before they sold out unicorn normcore crucifix brooklyn biodiesel lo-fi kinfolk wayfarers next level ramps craft beer. Blue bottle health goth squid authentic. Mumblecore you probably haven't heard of them tumeric enamel pin YOLO chicharrones farm-to-table +1 glossier tbh blog. Sartorial mixtape normcore, farm-to-table dreamcatcher irony cronut whatever squid tumblr 90's. Blog green juice franzen yr jianbing quinoa chicharrones tousled messenger bag cray kitsch drinking vinegar pour-over truffaut flexitarian. Schlitz shabby chic roof party asymmetrical listicle cloud bread vegan. Flannel mustache freegan street art bespoke put a bird on it. Banh mi skateboard before they sold out hella."
    excerpt="Try-hard edison bulb slow-carb pitchfork tumeric yr master cleanse health goth actually cred shoreditch. Enamel pin before they sold out unicorn normcore crucifix brooklyn biodiesel lo-fi kinfolk wayfarers next level ramps craft beer. Blue bottle health goth squid authentic."
    published_at="2017-10-22"
    slug="sample-post-4"
    title="Sample Post 4"
  />
  <PostSummary
    author="Matt Smrke"
    content="Pour-over snackwave mumblecore gochujang flexitarian fanny pack ugh cred actually sartorial cliche bespoke art party helvetica. Ethical raclette meggings, vice intelligentsia pok pok shoreditch synth meh. Shoreditch succulents aesthetic chicharrones iceland gastropub, VHS fam everyday carry actually. Trust fund wayfarers street art scenester. Photo booth humblebrag pork belly, hashtag brooklyn wayfarers bicycle rights before they sold out yuccie chia bushwick. Portland forage lomo edison bulb intelligentsia fam ugh chartreuse banh mi fixie prism master cleanse kinfolk polaroid. Truffaut readymade DIY 8-bit. Chambray fam pabst mustache. Locavore kombucha typewriter, four loko 90's fanny pack la croix. Lo-fi cliche bitters mustache. Blog snackwave yuccie, microdosing waistcoat single-origin coffee lumbersexual everyday carry cronut fashion axe tacos intelligentsia air plant subway tile vinyl. Vice organic next level prism salvia, hell of craft beer dreamcatcher wayfarers 8-bit keffiyeh migas live-edge blog pok pok. Tilde godard air plant vexillologist shaman organic bicycle rights pug, XOXO man bun vape butcher pitchfork yuccie. Iceland cray tumblr authentic, lumbersexual waistcoat gastropub you probably haven't heard of them street art cronut tumeric meh coloring book humblebrag bicycle rights. Pickled narwhal typewriter YOLO chartreuse messenger bag.
  
  Letterpress live-edge photo booth tbh, tattooed affogato hexagon small batch palo santo. Semiotics truffaut small batch, synth fashion axe actually unicorn chicharrones hoodie vape. La croix chambray meh gluten-free etsy cred vegan slow-carb trust fund seitan disrupt cardigan umami. Craft beer cliche skateboard polaroid bushwick selvage shoreditch trust fund seitan raclette 3 wolf moon crucifix. Next level single-origin coffee palo santo man braid letterpress enamel pin. Adaptogen copper mug cornhole, keffiyeh flannel franzen hammock stumptown raw denim af readymade cloud bread tote bag chicharrones. Kombucha YOLO aesthetic snackwave pop-up retro, pour-over kogi bushwick tilde kitsch helvetica. Disrupt master cleanse four dollar toast mustache. Man braid authentic swag taiyaki affogato, cred beard DIY lomo squid jianbing. Four dollar toast shoreditch artisan beard, street art farm-to-table locavore poutine umami live-edge whatever ramps."
    excerpt="Pour-over snackwave mumblecore gochujang flexitarian fanny pack ugh cred actually sartorial cliche bespoke art party helvetica. Ethical raclette meggings, vice intelligentsia pok pok shoreditch synth meh. Shoreditch succulents aesthetic chicharrones iceland gastropub, VHS fam everyday carry actually."
    published_at="2017-10-22"
    slug="sample-post-5"
    title="Sample Post 5"
  />
</div>
`;
